### ISSUE TITLE
Change model refer to missing model

### ISSUE THREAD

**Issue by TheoLeCalvar** [ATL-Research/incremental-class2relational#16]
We have an issue with the change model, they refer model element using relative path to XMI (see [here](https://github.com/ATL-Research/incremental-class2relational/blob/a24f02de57a0f2d8a9a74a3e4ca8f5eeda2f812f/models/SampleClassChangeModel.xmi#L8) which will break if files are moved
In TTC2018 change models used a different syntax which was working independently of file path (see [here](https://github.com/TransformationToolContest/ttc2018liveContest/blob/7900b13dc390d0d8f38e45845e17777ab3a550d9/models/1/change01.xmi#L10)).

My guess is that a save option is required to tell EMF to serialize these link in a certain way

**Comment by TheoLeCalvar** [comment]
If not possible path in references should be corrected once we converge on a folder structure

**Comment by sandragreiner** [comment]
yes I see...
If we apply the directory structure as discussed: 
        srcmodel + all its change models in one directory 
and I generate them in that way, it should work for the moment, doesn't it?

or is it currently crashing for you?

**Comment by TheoLeCalvar** [comment]
Yes, it should work. In the worst case we can manually patch then by updating the wrong path with a simple search / replace.
That's what I did on my machine to test the change model
